src/uu/split/src/split.rs

```diff
@@ -125,28 +127,50 @@ pub fn uu_app() -> Command {
         )
         .arg(
             Arg::new(OPT_NUMERIC_SUFFIXES)
-                .short('d')
                 .long(OPT_NUMERIC_SUFFIXES)
+                .require_equals(true)
                 .default_missing_value("0")
                 .num_args(0..=1)
+                .overrides_with(OPT_NUMERIC_SUFFIXES)
                 .help("use numeric suffixes instead of alphabetic"),
         )
         .arg(
-            Arg::new(OPT_SUFFIX_LENGTH)
-                .short('a')
-                .long(OPT_SUFFIX_LENGTH)
-                .value_name("N")
-                .default_value(OPT_DEFAULT_SUFFIX_LENGTH)
-                .help("use suffixes of fixed length N. 0 implies dynamic length."),
+            Arg::new(OPT_NUMERIC_SUFFIXES_SHORT)
+                .short('d')
+                .action(clap::ArgAction::SetTrue)
+                .value_parser(|_: &str| -> Result<String,clap::Error> {
+                    Ok("0".to_string()) // force value to "0"
+                })
+                .overrides_with(OPT_NUMERIC_SUFFIXES_SHORT)
+                .help("use numeric suffixes instead of alphabetic"),
```

Comment:
Maybe it's nice to say something like "like --numeric-suffixes but does not accept an argument" to explain why there are two arguments that do the same thing.

Reply:
not sure I understood the suggestion, could you please exand on that?

Reply:
The suggestion is to put a bit of text in the `help` string saying that it's the same as `--numeric-suffixes`. If I wouldn't know about these arguments and I would see two arguments with the same description in `--help`, I'd be a bit confused. Does that make sense?

Reply:
yep, that makes sense, thanks. Will update it

Reply:
Followed exact same order and wording as in GNU split help:
  -d     use numeric suffixes starting at 0, not alphabetic

   --numeric-suffixes[=FROM]
          same as -d, but allow setting the start value

   -x     use hex suffixes starting at 0, not alphabetic

   --hex-suffixes[=FROM]
          same as -x, but allow setting the start value

Reply:
Perfect!